Languages/English/Keyed/Keys.xml

```diff
@@ -1,29 +1,29 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <LanguageData>
 
-	<koisama.Numbers.Title>Numbers overview</koisama.Numbers.Title>
-	<koisama.Numbers.Count>Row Count</koisama.Numbers.Count>
-	<koisama.Numbers.ClickToToggle>Click to select pawn type.</koisama.Numbers.ClickToToggle>
+	<koisama.Numbers.Title>Numbers!</koisama.Numbers.Title>
+	<koisama.Numbers.Count>Number of rows</koisama.Numbers.Count>
+	<koisama.Numbers.ClickToToggle>Click to select metric or pawn type.</koisama.Numbers.ClickToToggle>
```

Comment:
It's only used as a tooltip on the Pawn toggle, not on any of the other buttons.

Maybe a redesign of "Click to {0} {1}." with {0} being either "select" if {1} is a pawn type, and "add" if {1} is any of the other buttons? Thoughts appreciated.